package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "react-responsive-components",
-  "version": "0.4.5",
+  "version": "0.5.0",
```

Comment:
No breaking changes on a major version upgrade of React and TS?

Reply:
Functionality wise we haven't noticed anything so far. It is only the typing.